src/Commands/Purview/GetRetentionLabel.cs

```diff
@@ -0,0 +1,25 @@
+﻿using PnP.PowerShell.Commands.Base;
+using PnP.PowerShell.Commands.Base.PipeBinds;
+using PnP.PowerShell.Commands.Utilities.REST;
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace PnP.PowerShell.Commands.Purview
+{
+    [Cmdlet(VerbsCommon.Get, "PnPRetentionLabel")]
+    [OutputType(typeof(IEnumerable<Model.Graph.Purview.RetentionLabel>))]
+    [OutputType(typeof(Model.Graph.Purview.RetentionLabel))]
```

Comment:
I guess we can remove this since it will be a collection ?

Reply:
@gautamdsheth : planning to add parameter Id to retrieve retention label by id , would that affect the function if one record is returned ? ..shall I add the parameter Id in the current PR?  I have kept the initial implementation to get retention labels simple as was not sure what I did met PnP PowerShell standards.

Reply:
Have a question  - do you see the need for this cmdlet ? As in, would be using it in frequently in near future ? 

Really don't want to add more cmdlets, if not needed , since we have quite a few. Would rather focus on bug fixes and code quality. Gets difficult to maintain. Also, since this is beta , there's a good chance MS will break it.

Reply:
PnP PowerShell has cmdlets to apply and get retention labels at site/list level 
Set-PnPLabel -List "<list_identifier>" -Label "Retain 10 years"
It would be great to have ability to manage retention labels within PnP PowerShell. I understand the concern of it being in still in beta and MS breaking it in the future. 
I would use it if it is within PnP PowerShell but can't tell about general usage by end users.

Reply:
Ok fair point 😊
Yeah, would be good to have Identity parameter as well in this one.